ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/rewriter/RemoveAndBypassNode.scala

```diff
@@ -0,0 +1,68 @@
+package pl.touk.nussknacker.restmodel.displayedgraph.rewriter
+
+import pl.touk.nussknacker.engine.graph.node.{NodeData, Sink}
+import pl.touk.nussknacker.restmodel.displayedgraph.{DisplayableProcess, ProcessRewriter}
+import pl.touk.nussknacker.restmodel.displayedgraph.displayablenode.Edge
+import pl.touk.nussknacker.restmodel.displayedgraph.displayablenode.EdgeType.FilterFalse
+
+
+object RemoveAndBypassNode {
+  private case class NodeEdges(incoming: List[Edge], outgoing: List[Edge])
+}
+
+class RemoveAndBypassNode(process: DisplayableProcess, node: NodeData) {
```

Comment:
I think that moving `process` to arguments of `apply` will better show how it is processed (where it is used). I see that `node` is really used "globally" by this class so it is in correct place. Also we can think about renaming `apply` to more expressive name e.g. `RemoveAndBypassNode(node).in(process)`. WDYT?